### ISSUE TITLE
[Socket]  [war]([Errno -2] Name or service not known)

### ISSUE THREAD

**Issue by souli0** [PowerScript/KatanaFramework#32]
[run] The module was launched...
   [inf] Wed Feb 20 17:39:16 2019
   [inf] Loading file [files/db/commons-dir-admin.tbl]
   [stp] [1] Step : Starting Brute Force...
 [Socket]  [war]([Errno -2] Name or service not known)

any idea how to fix it? i'm starting to overburn a bit a new point of view could help me thanks.

**Comment by UcanInek691** [comment]
did you fix that problem?

**Comment by Artook10** [comment]
Yes but can not remember exactly how. I had to change a search option from
true to false but not sure what it was now.

**Comment by UcanInek691** [comment]
oh, i think i got you. thanks :)))

**Comment by Artook10** [comment]
No problem wish I could have been more helpful.
